presto-main/src/main/java/com/facebook/presto/security/ReadOnlySystemAccessControl.java

```diff
@@ -110,15 +110,4 @@ public void checkCanShowSchemas(Identity identity, String catalogName)
     public void checkCanShowTablesMetadata(Identity identity, CatalogSchemaName schema)
     {
     }
-
```

Comment:
same for this one.

Reply:
Recovered in `Add access control checks for SHOW ROLES`